### DISCUSSION TITLE
Please add yandex map in expo

### DISCUSSION THREAD

**Discussion by dimaokk** [expo/expo#18508] [Proposals and Ideas]
Please add yandex map in expo https://github.com/volga-volga/react-native-yamap

**Comment by KAMRONBEK** [comment]
@dimaokk I build a maintained alternative on the Expo Modules API — [expo-yandex-mapkit](https://github.com/softwhere-uz/expo-yandex-mapkit) — CI-tested to compile against current React Native / New Architecture (including a bare-RN 0.86 consumer build) on both platforms. Leaving it as an option in case it helps. (Disclosure: I'm the author.)
